### PR TITLE
[zh-cn] sync less_than operator

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Get rid of untranslated pages.

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/operators/less_than/index.md [hunk 1]
```diff
@@ -1,104 +1,139 @@
...
 
-  - Boolean values `true` and `false` are converted to 1 and 0 respectively.
-  - `null` is converted to 0.
-  - `undefined` is converted to `NaN`.
-  - Strings are converted based on the values they contain, and are converted as `NaN` if they do not contain numeric values.
+其他运算符，包括 [`>`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than)、[`>=`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than_or_equal) 和 [`<=`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Less_than_or_equal) 使用与 `<` 相同的算法。有两种情况，所有四个运算符都返回 `false`：
 
-- If either value is [`NaN`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/NaN), the operator returns `false`.
-- Otherwise the values are compared as numeric values.
+- 如果其中一个操作数被转换为 BigInt，而另一个被转换为无法转换为 BigInt 值的字符串（当传递给 [`BigInt()`](/zh-CN/docs/Web/JavaScript/Reference/Errors/Invalid_BigInt_syntax)时，它会抛出[语法错误](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/BigInt)）。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/operators/less_than/index.md:34]:
```suggestion
- 如果其中一个操作数被转换为 BigInt，而另一个被转换为无法转换为 BigInt 值的字符串（当传递给 [`BigInt()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/BigInt)时，它会抛出[语法错误](/zh-CN/docs/Web/JavaScript/Reference/Errors/Invalid_BigInt_syntax)）。
```

File: files/zh-cn/web/javascript/reference/operators/less_than/index.md [hunk 2]
```diff
@@ -1,104 +1,139 @@
...
-- Otherwise JavaScript attempts to convert non-numeric types to numeric values:
+首先，通过依次调用其 [`[@@toPrimitive]()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toPrimitive)、[`valueOf()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Objects/valueOf)、[`toString()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Objects/toString) 方法，将对象[转换为原始类型](/zh-CN/docs/Web/JavaScript/Data_structures#强制原始值转换)（以 `"number"` 作为提示）。左边的操作数总是在右边的操作数之前被强制转换。请注意，尽管 `[@@toPrimitive]()` 被调用时带有 `"number"` 的提示（意味着有一点倾向于将对象变成数字），但返回值并没有[转换为数字](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)，因为字符串仍然被特别处理。
+- 如果两个值都是字符串，则根据它们所包含的 Unicode 码位的值，将它们作为字符串进行比较。
+- 否则，JavaScript 会尝试将非数值类型转化为数值类型：
+  - 布尔值 `true` 和 `false` 分别转化为 1 和 0。
+  - `null` 转化为 0。
+  - `undefined` 转化为 `NaN`。
+  - 字符串根据其包含的值进行转换，如果不包含数字值，则转换为 `NaN`。
+- 如果任意一个值为 [`NaN`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/NaN)，则运算符返回 `false`。
+- 否则，这些值将作为数字值进行比较。BigInt 和数字值可以一起比较
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/operators/less_than/index.md:30]:
```suggestion
- 否则，这些值将作为数值进行比较。BigInt 和数值可以一起比较。
```

File: files/zh-cn/web/javascript/reference/operators/less_than/index.md [hunk 3]
```diff
@@ -1,104 +1,140 @@
...
-## Description
+## 描述
 
-The operands are compared using the [Abstract Relational Comparison](https://tc39.es/ecma262/#sec-abstract-relational-comparison) algorithm, which is roughly summarised below:
+操作数经过多轮强制比较，可以总结为以下几点：
 
-- First, objects are converted to primitives using [`Symbol.ToPrimitive`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toPrimitive) with the `hint` parameter be `'number'`.
-- If both values are strings, they are compared as strings, based on the values of the Unicode code points they contain.
-- Otherwise JavaScript attempts to convert non-numeric types to numeric values:
+首先，通过依次调用其 [`[@@toPrimitive]()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toPrimitive)、[`valueOf()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Objects/valueOf)、[`toString()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Objects/toString) 方法，将对象[转换为原始类型](/zh-CN/docs/Web/JavaScript/Data_structures#强制原始值转换)（以 `"number"` 作为提示）。左边的操作数总是在右边的操作数之前被强制转换。请注意，尽管 `[@@toPrimitive]()` 被调用时带有 `"number"` 的提示（意味着有一点倾向于将对象变成数字），但返回值并没有[转换为数字](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)，因为字符串仍然被特别处理。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/operators/less_than/index.md:22]:
```suggestion
首先，通过依次调用其 [`[@@toPrimitive]()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toPrimitive)、[`valueOf()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/valueOf)、[`toString()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/toString) 方法，将对象[转换为原始类型](/zh-CN/docs/Web/JavaScript/Data_structures#强制原始值转换)（以 `"number"` 作为提示）。左边的操作数总是在右边的操作数之前被强制转换。请注意，尽管 `[@@toPrimitive]()` 被调用时带有 `"number"` 的提示（意味着有一点倾向于将对象变成数字），但返回值并没有[转换为数字](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)，因为字符串仍然被特别处理。
```

File: files/zh-cn/web/javascript/reference/operators/less_than/index.md [hunk 4]
```diff
@@ -1,104 +1,140 @@
...
 
-## See also
+## 参见
 
-- [Greater than operator](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than)
-- [Greater than or equal operator](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than_or_equal)
-- [Less than or equal operator](/zh-CN/docs/Web/JavaScript/Reference/Operators/Less_than_or_equal)
+- [大于操作符](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than)
+- [大于等于操作符](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than_or_equal)
+- [小于等于操作符](/zh-CN/docs/Web/JavaScript/Reference/Operators/Less_than_or_equal)
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/operators/less_than/index.md:140]: 运算符

File: files/zh-cn/web/javascript/reference/operators/less_than/index.md [hunk 5]
```diff
@@ -1,104 +1,140 @@
...
-- If either value is [`NaN`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/NaN), the operator returns `false`.
-- Otherwise the values are compared as numeric values.
+其他运算符，包括 [`>`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than)、[`>=`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than_or_equal) 和 [`<=`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Less_than_or_equal) 使用与 `<` 相同的算法。有两种情况，所有四个运算符都返回 `false`：
 
-## Examples
+- 如果其中一个操作数被转换为 BigInt，而另一个被转换为无法转换为 BigInt 值的字符串（当传递给 [`BigInt()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/BigInt)时，它会抛出[语法错误](/zh-CN/docs/Web/JavaScript/Reference/Errors/Invalid_BigInt_syntax)）。
+- 如果其中一个操作数被转化为 `NaN`，如不能转化为数字的字符串，或 `undefined`。
 
-### String to string comparison
+对于所有其他情况，这四种运算符有以下关系：
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/operators/less_than/index.md:38]: 其它

File: files/zh-cn/web/javascript/reference/operators/less_than/index.md (hunk 4 above)
Comment (jasonren0403) [reply files/zh-cn/web/javascript/reference/operators/less_than/index.md:140]:
```suggestion
- [大于运算符](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than)
- [大于等于运算符](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than_or_equal)
- [小于等于运算符](/zh-CN/docs/Web/JavaScript/Reference/Operators/Less_than_or_equal)
```

File: files/zh-cn/web/javascript/reference/operators/less_than/index.md (hunk 5 above)
Comment (jasonren0403) [reply files/zh-cn/web/javascript/reference/operators/less_than/index.md:38]:
```suggestion
对于所有其它情况，这四种运算符有以下关系：
```

File: files/zh-cn/web/javascript/reference/operators/less_than/index.md [hunk 6]
```diff
@@ -1,104 +1,140 @@
...
-## Description
+## 描述
 
-The operands are compared using the [Abstract Relational Comparison](https://tc39.es/ecma262/#sec-abstract-relational-comparison) algorithm, which is roughly summarised below:
+操作数经过多轮强制比较，可以总结为以下几点：
 
-- First, objects are converted to primitives using [`Symbol.ToPrimitive`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toPrimitive) with the `hint` parameter be `'number'`.
-- If both values are strings, they are compared as strings, based on the values of the Unicode code points they contain.
-- Otherwise JavaScript attempts to convert non-numeric types to numeric values:
+首先，通过依次调用其 [`[@@toPrimitive]()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toPrimitive)、[`valueOf()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/valueOf)、[`toString()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/toString) 方法，将对象[转换为原始类型](/zh-CN/docs/Web/JavaScript/Data_structures#强制原始值转换)（以 `"number"` 作为提示）。左边的操作数总是在右边的操作数之前被强制转换。请注意，尽管 `[@@toPrimitive]()` 被调用时带有 `"number"` 的提示（意味着有一点倾向于将对象变成数字），但返回值并没有[转换为数字](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)，因为字符串仍然被特别处理。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/operators/less_than/index.md:22]:
```suggestion
首先，通过依次调用其 [`[@@toPrimitive]()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toPrimitive)（以 `"number"` 作为提示）、[`valueOf()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/valueOf) 和 [`toString()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/toString) 方法，将对象[转换为原始类型](/zh-CN/docs/Web/JavaScript/Data_structures#强制原始值转换)。左边的操作数总是在右边的操作数之前被强制转换。请注意，尽管 `[@@toPrimitive]()` 被调用时带有 `"number"` 的提示（意味着有一点倾向于将对象变成数字），但返回值并没有[转换为数字](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#number_强制转换)，因为字符串仍然被特别处理。
```

File: files/zh-cn/web/javascript/reference/operators/less_than/index.md [hunk 7]
```diff
@@ -1,104 +1,140 @@
...
+  - 布尔值 `true` 和 `false` 分别转化为 1 和 0。
+  - `null` 转化为 0。
+  - `undefined` 转化为 `NaN`。
+  - 字符串根据其包含的值进行转换，如果不包含数字值，则转换为 `NaN`。
+- 如果任意一个值为 [`NaN`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/NaN)，则运算符返回 `false`。
+- 否则，这些值将作为数值进行比较。BigInt 和数值可以一起比较。
 
-- If either value is [`NaN`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/NaN), the operator returns `false`.
-- Otherwise the values are compared as numeric values.
+其他运算符，包括 [`>`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than)、[`>=`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than_or_equal) 和 [`<=`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Less_than_or_equal) 使用与 `<` 相同的算法。有两种情况，所有四个运算符都返回 `false`：
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/operators/less_than/index.md:33]:
```suggestion
其它运算符，包括 [`>`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than)、[`>=`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Greater_than_or_equal) 和 [`<=`](/zh-CN/docs/Web/JavaScript/Reference/Operators/Less_than_or_equal) 使用与 `<` 相同的算法。有两种情况，所有四个运算符都返回 `false`：
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**